.github/workflows/dependency-review.yml

```diff
@@ -1,5 +1,12 @@
 name: Dependency Review
-on: [pull_request]  # yamllint disable-line rule:truthy
+on:
+  pull_request:
+    # yamllint disable-line rule:truthy
```

Comment:
🔎 This will need to stay on the same line as `on:` in order to be honored by YAMLLint 🙂 

```suggestion
on: # yamllint disable-line rule:truthy
  pull_request:
```

Reply:
Thanks, and done